deepspeed/runtime/config.py

```diff
@@ -446,6 +446,8 @@ def get_pipeline_config(param_dict):
         "partition": "best",
         "seed_layers": False,
         "activation_checkpoint_interval": 0,
+        "pipe_partitioned": "auto",
```

Comment:
Why use "auto" as a value to represent True rather than simply make this a boolean config?

Reply:
@tjruwase agreed, see updated change